generated/niswitch/setup.py

```diff
@@ -51,7 +51,7 @@ def read_contents(file_to_read):
     tests_require=['pytest'],
     test_suite='tests',
     classifiers=[
-        "Development Status :: 4 - Beta",
+        "Development Status :: 3 - Alpha",
```

Comment:
I'm confused... This should not say Alpha.

Reply:
Yes, it should because the current version is X.Y.Z.dev1. Once we go to release to PyPI, .dev1 will go away and this will be updated to Stable.

I updated the comments and CHANGELOG.md to address this but I hadn't updated the PR with this level of granularity. Added now.